hoth_api.yaml

```diff
@@ -407,10 +407,77 @@ components:
           type: string
           enum:
             - DESCRIPTION_OF_WORK
+            - INDEPENDENT_GOVERNMENT_COST_ESTIMATE
         templatePayload:
           type: object
           oneOf:
             - $ref: "#/components/schemas/DescriptionOfWork"
+            - $ref: "#/components/schemas/IndependentGovernmentCostEstimate"
+    IndependentGovernmentCostEstimate:
+      description: "Data used for generating the Independence Government Cost Estimate (IGCE) document"
+      type: object
+      properties:
+        funding_document:
+          description: >-
+            Information related to a funding plan. Either both the GTC number and Order 
+            number or only the MIPR number. 
+          type: object
+          properties:
+            funding_type:
+              type: string
+              enum: 
+                - FS_FORM
+                - MIPR
+            gtc_number: 
+              type: string
+            order_number:
+              type: string
+            mipr_number: 
+              type: string
+        surge_capabilities:
+          type: string
+        periods_estimate:
+          type: array
+          items:
+            $ref: "#/components/schemas/PeriodEstimate"
+    PeriodEstimate:
+      description: "Information specific to a base or option period sheet in the IGCE document"
+      type: object
+      properties:
+        period:
+          description: "The period of performance for a specific period sheet."
+          $ref: "#/components/schemas/Period"
```

Comment:
We should use:
```
    PeriodOfPerformance:
      description: "Part of DOW, represents a single Period of Performance"
      type: object
      properties:
        base_period:
          $ref: "#/components/schemas/Period"
        option_periods:
          type: array
          items:
            $ref: "#/components/schemas/Period"
        pop_start_request:
          type: boolean
        requested_pop_start_date:
          type: string
          format: date
          example: "2021-07-01"
        time_frame:
          type: string
          enum:
            - NO_LATER_THAN
            - NO_SOONER_THAN
        recurring_requirement:
          type: boolean
```

Reply:
Specifically for 
```
 base_period:
          $ref: "#/components/schemas/Period"
        option_periods:
          type: array
          items:
            $ref: "#/components/schemas/Period"
```

Reply:
My concern is ordering the Periods on the backend

Reply:
We would be able to identify the base period using `period_type` and then for all the option periods we can use the `option_order` to order on the backend. I'd be open to using the PoP schema, but I'd be concerned the extra data might cause validation issues.